OBJC_INTEROP.md

```diff
@@ -0,0 +1,124 @@
+# _Kotlin/Native_ interoperability with Swift/Objective-C
+
+This documents covers some details of Kotlin/Native interoperability with
+Swift/Objective-C.
```

Comment:
Shouldn't we use some common name for Swift and Objective-C?

Reply:
Good idea, although I've no idea what could it be.